.github/workflows/release.yaml

```diff
@@ -259,6 +259,8 @@ jobs:
         uses: actions/checkout@v3
       - run: git push origin HEAD:web --force
 
-  push-devcontainer-base-image:
+  publish-devcontainer:
```

Comment:
I don't know if the word "publish" is appropriate here; I think "push" is more common.

Reply:
Yeah, I thought it was consistent with the rest of the workflow, but agree re docker conventions, will change